audio/generaluser-soundfont/Portfile

```diff
@@ -3,7 +3,8 @@
 PortSystem          1.0
 
 name                generaluser-soundfont
-version             1.47
+version             1.471
+revision            0
```

Comment:
`revision` is only used for non-zero values.

Reply:
Or when a port has subports, to clearly identify which subport is at which revision. But this port does not use subports.